bon/tests/integration/ui/compile_fail/map_macro.stderr

```diff
@@ -0,0 +1,13 @@
+error: duplicate map key
+  --> tests/integration/ui/compile_fail/map_macro.rs:14:9
+   |
+14 |         "Hello": "Mars",
+   |         ^^^^^^^
+
+warning: unused variable: `strings`
+ --> tests/integration/ui/compile_fail/map_macro.rs:5:9
+  |
+5 |     let strings: HashMap<u64, String> = map! {
+  |         ^^^^^^^ help: if this is intentional, prefix it with an underscore: `_strings`
+  |
+  = note: `#[warn(unused_variables)]` on by default
```

Comment:
Let's add `_` to the variable name to avoid having this warningin the stderr snapshot

Reply:
Fixed by adding a `compile_fail` doctest instead.